gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/ChannelPool.java

```diff
@@ -152,6 +152,13 @@ Channel getChannel(int affinity) {
   /** {@inheritDoc} */
   @Override
   public ManagedChannel shutdown() {
+    if (LOG.isLoggable(Level.FINE)) {
+      // Synthetic exception to generate a stacktrace in the logs
+      RuntimeException e =
+          new RuntimeException("Initiating graceful shutdown due to explicit request");
+      LOG.log(Level.FINE, e.getMessage(), e);
```

Comment:
What's wrong with just logging a message without a synthetic exception?

Reply:
Interpreting from the PR description: passing a Throwable to `log` will print the stacktrace - thus showing information about why the shutdown occurred not just that it's shutting down.

For a rough equivalent w/o passing the Throwable into the log and triggering Error Reporting, consider something like:
```java
String stackTrace = Arrays.stream((new Throwable()).getStackTrace())
    .map(StackTraceElement::toString)
    .collect(Collectors.joining(System.lineSeparator() + "  "));
```

Reply:
removed stacktrace